app/views/bookings/index.html.erb

```diff
@@ -5,18 +5,12 @@
     <% @bookings.each do |booking| %>
       <% if booking.user == current_user %>
        <%# this if checks that the user is the person booking the dress %>
-      <% booked_dress = Dress.find(booking.dress_id) %>
+      <% booked_dress = booking.dress %>
         <div class="dress-card text-center mb-3">
-          <%= link_to booking_path(booking) do %>
-            <%# Image tag to be updated with cloudinary %>
-            <%= booked_dress.photos.attached? ? cl_image_tag(booked_dress.photos.first.key) :
```

Comment:
Maybe we could do booking.dress.photos.attached here and booking.dress.photos.first.key instead of booked_dress. I know its long but I think we aren't supposed to set variables in the view :/

Reply:
photo taken out as we'll move the bookings into the profile bookings. Probably in a table or list format

Reply:
This was old code from the dress_index page, it mightve been updated already too